src/ground_truth_nsrts.py

```diff
@@ -2172,6 +2172,7 @@ def _get_predicate(base_pred_name: str,
     op_name_count_pick = itertools.count()
     op_name_count_place = itertools.count()
 
+    assert CFG.option_learner == "no_learning"
```

Comment:
don't we need a similar assert in `get_options_by_names`? but this would break data collection. maybe we shouldn't have these asserts at all actually?

Reply:
good point, i'll remove